src/app/find/locationDetails/locationDetails.css

```diff
@@ -18,24 +18,37 @@
 .categoryCard {
```

Comment:
How come any styles for the existing pages had to change (except where it's related to adding the button at the bottom)?
Or were some of these changes merge issues or something?